test/TestInlineSetup.pm

```diff
@@ -1,8 +1,7 @@
 use strict; use warnings; use diagnostics;
 package TestInlineSetup;
 
-use File::Path;
-use File::Spec;
+use File::Temp 0.19;
```

Comment:
Adding File::Temp is a great idea, but removing File::Spec seems incorrect.

Reply:
Now File::Spec is not used by any piece of code in TestInlineSetup.pm. Hence I cannot see reason why to keep it loading it from there.

Reply:
Fixed in 003efa7f1855340fd6b4317ac7f1dee7bb895040.